adflow/mphys/mphys_adflow.py

```diff
@@ -470,8 +470,10 @@ def solve_linear(self, d_outputs, d_residuals, mode):
         if mode == "fwd":
             d_outputs["adflow_states"] = solver.solveDirectForRHS(d_residuals["adflow_states"])
         elif mode == "rev":
+            phi = np.zeros_like(d_residuals["adflow_states"])
             # d_residuals['adflow_states'] = solver.solveAdjointForRHS(d_outputs['adflow_states'])
-            solver.adflow.adjointapi.solveadjoint(d_outputs["adflow_states"], d_residuals["adflow_states"], True)
+            solver.adflow.adjointapi.solveadjoint(d_outputs["adflow_states"], phi, True)
```

Comment:
Remove the commented out line if it's not needed

Reply:
Fixed it.